ue5osc/__init__.py

```diff
@@ -4,6 +4,13 @@
 from pythonosc.osc_server import BlockingOSCUDPServer
 
 from ue5osc.osc_dispatcher import OSCMessageReceiver
+from enum import IntEnum
+
+
+class ObjectType(IntEnum):

```

Comment:
Rename (I gave an alternative in boxnav, but feel free to change to something else.